packages/manager/apps/dedicated/client/app/ip/components/list/block/move/ip-ip-block-move.html

```diff
@@ -114,6 +114,77 @@
                 ></p>
             </div>
 
+            <div data-ng-show="!loading.save && data.ipBlock.isAdditionalIpv6">
+                <div data-ng-show="loading.ipCanBeMovedTo">
+                    <p>
+                        <span
+                            data-translate="vrack_add_ipv6_modal_description"
+                            data-translate-values="{ ipblock: '<strong>' + data.ipBlock.ipBlock + '</strong>', vrack: '<strong>' + model.serviceName.service + '</strong>', region: '<strong>' + data.ipBlock.region + '</strong>' }"
+                        ></span>
+                    </p>
+                    <p>
+                        <span
+                            data-translate="vrack_add_ipv6_modal_description1"
+                        ></span>
+                    </p>
+                    <p>
+                        <span
+                            data-translate="vrack_add_ipv6_modal_description2"
+                            data-translate-values="{ region: '<strong>' + data.ipBlock.region + '</strong>' }"
+                        ></span>
+                    </p>
```

Comment:
**nitpick:** Redundant `span` can be avoided. Directly add `data-translate` on `p` elements